### PR TITLE
feat(mv-options): support mv options crud

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

**Comment by gagantrivedi** [comment]
Depends on https://github.com/Flagsmith/flagsmith/pull/1515

File: client_test.go [hunk 1]
```diff
@@ -513,3 +435,290 @@ func TestGetFeature(t *testing.T) {
...
+    "uuid": "8d3512d3-721a-4cae-9855-56c02cb0afe9",
+    "type": "unicode",
+    "string_value": "option_value_30",
+    "boolean_value": null,
+    "default_percentage_allocation": 60.0,
+    "feature": 1
+}
+`
+const MVFeatureOptionUUID = "8d3512d3-721a-4cae-9855-56c02cb0afe9"
+const MVFeatureOptionID int64 = 136
```
Comment (matthewelwell) [inline client_test.go:451]: Is this just an arbitrary number? If so, it seems a little odd to choose 136 - makes it seem like it's actually using some external source for data.

File: client_test.go [hunk 2]
```diff
@@ -513,3 +435,290 @@ func TestGetFeature(t *testing.T) {
...
+
+	assert.Equal(t, float64(60), featureMVOption.DefaultPercentageAllocation)
+	assert.Equal(t, FeatureID, *featureMVOption.FeatureID)
+	assert.Equal(t, FeatureUUID, featureMVOption.FeatureUUID)
+
+	assert.Equal(t, ProjectID, *featureMVOption.ProjectID)
+
+}
+
+func TestTestCreateFeatureMVOptionWithFeatureIDNotSet(t *testing.T) {
```
Comment (matthewelwell) [inline client_test.go:656]:
```suggestion
func TestCreateFeatureMVOptionWithFeatureIDNotSet(t *testing.T) {
```

File: client_test.go (hunk 1 above)
Comment (gagantrivedi) [reply client_test.go]: Yep, good catch

Comment (matthewelwell) [reply client_test.go]: Can we not use `1` ? `150` still seems a little specific to me?

Comment (gagantrivedi) [reply client_test.go]: 1 is already being used by `feature ID` I'd like to avoid using the same one, give me a bit more confidence

Comment (matthewelwell) [reply client_test.go]: Yep, fair point. Why not `2` then? 😄

Comment (gagantrivedi) [reply client_test.go]: Well, I am saving that for feature ID 2

Comment (matthewelwell) [reply client_test.go]: Ok, then leave it at 150 but maybe add a quick comment?

Comment (gagantrivedi) [reply client_test.go]: What if I make it 200?

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**